apps/studio/src/server/modules/resource/resource.service.ts

```diff
@@ -321,7 +348,11 @@ export const getLocalisedSitemap = async (
               ResourceType.Collection,
             ])
             .innerJoin("ancestors", "ancestors.parentId", "Resource.id")
-            .select(defaultResourceSelect),
+            .select(({ eb }) => [
+              eb.cast<string>(eb.val(""), "text").as("summary"),
+              eb.cast<string>(eb.val(""), "text").as("thumbnail"),
```

Comment:
why need to cast?

Reply:
ye, this is also abit odd! turns out, in pg 17 (my local version), you didnt need to. but in pg 15 (what `staging` was running), the type inference here fails on pg, so it needed the cast